design/Implemented/supporting-volumeattributes-resource-policy.md 

```diff
@@ -0,0 +1,84 @@
+# Adding Support For VolumeAttributes in Resource Policy
```

Comment:
Seems the design MD doc isn't rendered correctly
![image](https://github.com/user-attachments/assets/7a2209c6-6456-40dd-85e6-f160324c0584)

Reply:
Fixed this now.

![image](https://github.com/user-attachments/assets/f1d94795-3720-4a9d-8882-69af10f2ef6b)